templates/site/js/menu.js

```diff
@@ -21,7 +21,9 @@ document.addEventListener("DOMContentLoaded", function () {
     })
 
     technologyButton.addEventListener("click", () => {
-        bsOffcanvas.toggle();
+        if(window.window.innerWidth < 990){
```

Comment:
why not just:

```js
bsOffcanvas.hide();
```

So we don't have to keep track of screen widths?

Reply:
That's definitely a more encompassing solution.  Changed to uses hide.